Cargo.toml

```diff
@@ -1,89 +1,62 @@
-[package]
```

Comment:
Nit but let's format TOML files as well for consistency.

Reply:
Should I use https://github.com/tamasfe/taplo or smth else for formatting?

Reply:
Yes, that is a good choice. I'll add it to the upcoming `CONTRIBUTING.md`!